src/app/core/filter/filter/filter.component.html

```diff
@@ -0,0 +1,12 @@
+<app-list-filter
+  *ngFor="let filter of filterSelections"
+  [filterConfig]="filter.filterSettings"
+  [(selectedOption)]="filter.selectedOption"
+  [displayAsToggle]="filter.display !== 'dropdown'"
```

Comment:
I think we recently discovered that `displayAsToggle` is not implemented anymore. Let's remove it completely as part of this refactor?